### DISCUSSION TITLE
Настраиваемые папки сохранения файлов

### DISCUSSION THREAD

**Discussion by LostAlly** [totumonline/totum-ru-issues-and-discussions#511] [Неопределенное]
Можно ли как-то не внося изменений в исходники Тотума сделать настройку в таблице для хранения файлов отдельно от общей папки Тотума.
Пример того, что мне нужно
У таблицы заказы настройка - путь к папке
При создании заказа в таблице создается папка номер заказа по пути заданном пусть даже в коде действия.
Файлы заливаются в разные поля таблицы и в зависимости от поля создают и заливаются в подпапки папки заказа.
Если можно такое сделать, сколько это могло бы стоить?

**Comment by ttm-alexeypolunin** [comment]
Это можно сделать через SSH линукса, устанавливая ссылки на файлы ln -s

С ssh взяимодействует ф.действия execSSH. Кодом действия собираете и изменяете, что вам нужно.

Те файлы лежат там, где тотум положил, но еще есть структура с папками как вам нужно в которой лежат ссылки на эти файлы. Можно в этом случае даже человекочитаемые имена задавать.

**Reply by LostAlly** [reply]
Проблема в том, что с файлами будут работать напрямую в расшареной по сети папке, эта папка будет подмонтирована к серверу Тотума, но в основном обращаться буду к файлам напрямую для работы.
Нужно подумать конечно поглубже насчет вашего способа, но мне кажется это в любом случае похож костыль и было здорово иметь возможность более гибкого оперирования с путями хранения хотябы отдельно разработанными модулями или самим тотумом.

**Reply by ttm-alexeypolunin** [reply]
Мы точно не будем такого типа разрабатывать тк это очень специальная задача со стоимостью во много нулей.

Никаких проблем, что потом папка будет расшарена по сети и из нее будут читать данные тк линукса ссылка аналогична основному файлу.

Загружать они в эту папку и удалять из нее файлы, так, что бы оно подцепилось в Totum они не смогут, тк нужна обработка Тотумом.
